tasks/internal_reconcile_report_mismatch/schemas/input.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
We should discuss this at our architecture meeting, in the past we did an input schema which would require a POST in the body. We should really follow API design best practices, wherein we would pass this along int he query string as part of a GET. In addition, we should talk through paging as I feel this should be more robust. We may also want to talk about leveraging a GraphQL server with docker/ECS instead of a Lambda since in the end these reports will likely be displayed on some web page.

Reply:
I hesitate to add research/development/API deviations in a project that's already behind schedule.

Reply:
created https://bugs.earthdata.nasa.gov/browse/ORCA-413